lib/CoreDisTools/.nuget/runtime.json

```diff
@@ -1,18 +1,18 @@
 {
   "runtimes": {
     "win7-x64": {
```

Comment:
Same casing issue in all the files. The description might need some rewording... Otherwise, looks good to me.